src/pretalx/submission/models/question.py

```diff
@@ -61,6 +61,20 @@ class QuestionTarget(Choices):
     ]
 
 
+class QuestionRequired(Choices):
+    NONE = "none"
```

Comment:
Please always use these attributes for comparison in the rest of the code base. Comparing to `"require after"` is fragile when it would be better to compare to `QuestionRequired.REQUIRE_AFTER`.

Reply:
Fixed.